.travis.yml

```diff
@@ -9,21 +9,24 @@ addons:
 
 matrix:
   include:
-    # The normal tests
-    - jdk: oraclejdk8
-      os: linux
-      dist: trusty
-    - jdk: oraclejdk9
-      os: linux
-      dist: trusty
-    - os: osx
-      osx_image: xcode9.2
-
     # Check the format
     - stage: Check format
       jdk: oraclejdk8
       os: linux
       dist: trusty
+      install:
+        # do not use default install (runs formatter)
+        - mvn dependency:go-offline
       script:
-      - mvn test -B
-      - ./check_formatted.sh
\ No newline at end of file
+        - mvn -Pcheck-formatted validate formatter:validate
+
+    # The normal tests
+    - stage: Build + test
+      - jdk: oraclejdk8
```

Comment:
This line possibly contains a small problem for travis.
Because the `-` defines a new job inside the matrix.
Therefore we possibly have an empty job inside the `Build + Test` stage.
I suggest removing the `-` at the beginning of this line